### PR TITLE
Log requests that contain a route with transfer

### PR DESCRIPTION
**Pull request by kevinchan159**
Addresses #144 

I added a `containsTransfer` field to the LogUtils in `RouteRouter` which will tell us if the current route request returned a route w/ a transfer.

### CODE REVIEW

File: src/utils/RouteUtils.js [hunk 1]
```diff
@@ -25,6 +25,22 @@ async function isBusStop(location: string): Promise<boolean> {
...
+ *
+ * @param route
+ * @returns {boolean}
+ */
+function routeContainsTransfer(route: Object): boolean {
+  const { directions } = route;
+  const routeNumbers = [];
+  directions.forEach((direction) => {
+    const { routeNumber } = direction;
+    if (routeNumber !== null && !routeNumbers.includes(routeNumber)) routeNumbers.push(routeNumber);
```
Comment (meganle) [inline src/utils/RouteUtils.js]: could you just do `if (routeNumber &&...`

File: src/routers/v1/RouteRouter.js [hunk 2]
```diff
@@ -40,7 +40,8 @@ class RouteRouter extends ApplicationRouter<Array<Object>> {
         time,
         uid,
       };
-      LogUtils.log({ category: 'routeRequest', request });
+      const containsTransfer = routes.find(route => RouteUtils.routeContainsTransfer(route)) !== undefined;
+      LogUtils.log({ category: 'routeRequest', request, containsTransfer });
```
Comment (young-k) [inline src/routers/v1/RouteRouter.js]: we should only log if it actually contains a transfer, not if it doesn't (the majority of routes don't contain transfers)

Comment (kevinchan159) [reply src/routers/v1/RouteRouter.js]: well right now we log every request anyways so I just added a field to that log

Comment (young-k) [reply src/routers/v1/RouteRouter.js]: discussed offline, we should remove this log

**Pull request merged**